src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -173,13 +175,12 @@ def flexible_server_update_custom_func(cmd, instance,
                                        tags=None,
                                        auto_grow=None,
                                        assign_identity=False,
-                                       ha_enabled=None,
```

Comment:
Why remove this parameter? CX can't enable/disable ha anymore?

Reply:
Thanks evelyn. MySql doesn't support updating a regular server to HA server. This is to prevent misleading customers.